modules/openapi-generator-cli/src/test/resources/batch/jaxrs-datelib-j8-nested-include.yaml

```diff
@@ -0,0 +1,8 @@
+---
+"!include": common/jaxrs-datelib-j8.yaml
+generatorName: jaxrs-jersey
+inputSpec: batch/specs/petstore.yaml
```

Comment:
I wonder if we should move `inputSpec` from here to `jaxrs-datelib-j8-nested.yaml` to verify that the initial `!include` processing will also load sibling properties?